source/getting_started/on-vm.rst

```diff
@@ -95,121 +53,49 @@ caas-flashfiles-eng.<user>.zip
 Create a CaaS virtual disk
 **************************
 
-To create a virtual disk containing the CaaS partitions, you'll need
-a USB flash drive and at least 16GB free disk space. Follow the instructions
-below to create and set up CaaS partitions on a *qcow2* formatted virtual
-disk.
-
-#. Create a 16GB empty disk image of *qcow2* type:
-
-    .. code-block:: bash
-
-        $ qemu-img create -f qcow2 ~/caas/android.qcow2 16G
-
-#. Connect the USB flash drive to the development host.
-
-#. Run the :command:`lsusb` command. Use the output to identify the vendor
-   ID and product ID of the USB flash drive.
-   :
+.. note::
+        This section is for flashing *flashfiles* package, no action is required if the image caas.img is used.
```

Comment:
Skip this section if you plan to boot the device directly with the GPT disk image file:`caas.img`.